tests/integration/test_config_decryption/configs/config_invalid_chars.xml

```diff
@@ -0,0 +1,12 @@
+<clickhouse>
+    <encryption_codecs>
+        <aes_128_gcm_siv>
+            <key_hex>00112233445566778899aabbccddeeff</key_hex>
+        </aes_128_gcm_siv>
+        <aes_256_gcm_siv>
```

Comment:
We cannot combine two tests into one test. Because in each test an exception is thrown and there is server shutdown. So we cannot see both exceptions messages at the same time.